starter-code/src/clue.js

```diff
@@ -1,94 +1,259 @@
+    age: 62,
+    image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation: "Retired Football player"
+}   
 
 // Weapons
+const rope = {
+    name: "rope",
+    weight: 10
+}
+const knife = {
+    name: "knife",
+    weight: 8
+}
+const candlestick = {
+    name: "candlestick",
+    weight: 2
+}
+
+const dumbbell = {
+    name: "dumbbell",
+    weight: 30
+}
+ 
+const poison = {
+    name: "poison",
+    weight: 2
+}
+const axe = {
+    name: "axe",
+    weight: 15
+}
+const bat = {
+    name: "bat",
+    weight: 13
+}
+const trophy = {
+    name: "trophy",
+    weight: 25
+}
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+const pistol = {
+    name: "pistol",
+    weight: 20
+}
 
 // Rooms
-name: Dining Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+const diningRoom ={
+    name: "Dining Room"
+}
+const conservatory ={
+    name: "Conservatory"
+}
+const kitchen ={
+    name: "Kitchen"
+}
+const study ={
+    name: "Study"
+}
+const library ={
+    name: "Library"
+}
+const billiardRoom ={
+    name: "Billiard Room"
+}
+const lounge ={
+    name: "Lounge"
+}
+const ballroom ={
+    name: "Ballroom"
+}
+const hall ={
+    name: "Hall"
+}
+const spa ={
+    name: "Spa"
+}
+const livingRoom ={
+    name: "Living Room"
+}
+const observatory ={
+    name: "Observatory"
+}
+const theater ={
+    name: "Theater"
+}
+const guestHouse ={
+    name: "Guest House"
+}
+const patio ={
+    name: "Patio"
+}
 
 // Characters Collection
 const charactersArray = [];
 
+charactersArray.push(mrGreen);
+charactersArray.push(drOrchid);
+charactersArray.push(profPlum);
+charactersArray.push(missScarlet);
+charactersArray.push(mrsPeacock);
+charactersArray.push(mrMustard);
+
 // Rooms Collection
 const roomsArray = [];
 
+roomsArray.push(diningRoom);
+roomsArray.push(conservatory);
+roomsArray.push(kitchen);
+roomsArray.push(study);
+roomsArray.push(library);
+roomsArray.push(billiardRoom);
+roomsArray.push(lounge);
+roomsArray.push(ballroom);
+roomsArray.push(hall);
+roomsArray.push(spa);
+roomsArray.push(livingRoom);
+roomsArray.push(observatory);
+roomsArray.push(theater);
+roomsArray.push(guestHouse);
+roomsArray.push(patio);
+
 // Weapons Collection
 const weaponsArray = [];
+
+weaponsArray.push(rope);
+weaponsArray.push(knife);
+weaponsArray.push(candlestick);
+weaponsArray.push(dumbbell);
+weaponsArray.push(poison);
+weaponsArray.push(axe);
+weaponsArray.push(bat);
+weaponsArray.push(trophy);
+weaponsArray.push(pistol);
```

Comment:
Podéis, con un solo push por cada array, meter todos los argumentos que necesités